.buildkite/pipeline.yml

```diff
@@ -21,8 +21,15 @@ steps:
     env: *common_env
     plugins: *common_plugins
 
-  - label: Lints via Danger
-    command: .buildkite/commands/danger.sh
-    agents: *common_agents
-    env: *common_env
-    plugins: *common_plugins
+  - group: Linters
```

Comment:
The `group` results in this:

<img width="696" alt="image" src="https://user-images.githubusercontent.com/1218433/203904695-a4ed07f5-def6-400b-b2ef-5ed11da87193.png">